app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js

```diff
@@ -432,6 +432,13 @@ define([
             },
             $.mage.__('Please enter a valid phone number. For example (123) 456-7890 or 123-456-7890.')
         ],
+        'validate-phone': [
+            function (value) {
+                return utils.isEmptyNoTrim(value) ||
+                    /^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$/.test(value);
```

Comment:
According to my tests, the slash has to be escaped:
```suggestion
                    /^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\.\/0-9]*$/.test(value);
```

Reply:
I have fixed it. 
Thank You for advice